test_files/es6/arrow_fn.js

```diff
@@ -1 +1 @@
-var fn3 = (/** number */ a) => 12;
+var fn3 = (a) => 12;
```

Comment:
looks like we lost the comment? But it appears in sickle/arrow_fn.ts, so I am very confused.